### PR TITLE
Layout Slider: Responsive Height

### LINKED ISSUE #307: Layer Slider: Disable height setting or put in option to remove height on mobile devices
The layer slider has a serious problem in that if you set a height and then add multiple columns or a few rows to exceed this height limit, it won't display correctly on mobile devices as the other rows / columns will not be visible

[Here's a reference layout](https://u.teknik.io/m35Iv.json), the height is set very low to make it easier to see this issue without having to scroll for every.

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Fixed repo of #708 

----

This PR will resolve #307 as per Braam's suggestion:

> [...] add a separate height setting for mobile.

And:

> Leaving the mobile height setting blank should [...] have the same effect as disabling the [...] other height setting on mobile

### CODE REVIEW

File: widgets/layout-slider/styles/default.less [hunk 1]
```diff
@@ -56,4 +70,4 @@
 		}
 	}
 
-}
+}
```
Comment (braamgenis) [inline widgets/layout-slider/styles/default.less]: Newline please. :)

File: widgets/layout-slider/layout-slider.php [hunk 2]
```diff
@@ -311,4 +338,4 @@ function add_default_measurement_unit($val) {
 	}
 }
 
-siteorigin_widget_register('sow-layout-slider', __FILE__, 'SiteOrigin_Widget_LayoutSlider_Widget');
+siteorigin_widget_register('sow-layout-slider', __FILE__, 'SiteOrigin_Widget_LayoutSlider_Widget');
```
[the remark this answers is not in the document]
Comment (AlexGStapleton) [reply widgets/layout-slider/layout-slider.php]: Oh. Sorry about that!

File: widgets/layout-slider/layout-slider.php [hunk 3]
```diff
@@ -125,6 +125,12 @@ function get_widget_form(){
 						'default' => 'default',
 					),
 
+					'height_responsive' => array(
+						'type' => 'measurement',
+						'label' => __( 'Responsive Height', 'so-widgets-bundle' ),
+						'default' => 'default',
```
[the remark this answers is not in the document]
Comment (AlexGStapleton) [reply widgets/layout-slider/layout-slider.php]: Doesn't appear to be.

**Comment by AlexGStapleton** [comment]
How does that look?

Responsive Height for Hero widget was added in #657 
Really sorry about including the build. 😞 
I've installed the editor config extension for Sublime Text.

**Comment by Misplon** [comment]
Having worked on demos today, I can confirm the need for a mobile height setting in the Layout Slider. Thanks guys!

**Pull request merged**